proto/p4/config/p4info.proto

```diff
@@ -21,14 +21,12 @@ import "google/protobuf/any.proto";
 package p4.config;
 
 message P4Info {
-  repeated Table tables = 1;
-  repeated Action actions = 2;
-  repeated ActionProfile action_profiles = 3;
-  repeated Counter counters = 4;
-  repeated Meter meters = 5;
-
-  // for extern?
-  repeated google.protobuf.Any externs = 100;
+  repeated Extern externs = 1;
```

Comment:
So this was discussed in our Friday weekly. The idea was to avoid "Extern" eventually getting buried in the middle as more fields are added (improved readability), since the bottom will grow and the top is fixed.